content/ja/faq/add_script.md

```diff
@@ -7,7 +7,7 @@ keywords: スクリプト, コード, 作成, javascript, 挙動
 
 <img src="https://playcanvas.com/static-assets/instructions/add-new-script.gif"/>
 
-Javascriptを使用してエンティティの挙動を制御できます。スクリプトコンポーネントを追加して、新しいスクリプトを追加します。
+You can use javascript to control the behavior of entities. Add a script component, and create a new script.
```

Comment:
Why are translations being removed?

Reply:
Since you asked your question, this was fixed. However, it was because in the English, the word 'behaviour' was changed to 'behavior'. I then went into Transifex to say it's OK to use the original translation. It's not smart enough to know that though unfortunately.